routers/web/repo/issue.go

```diff
@@ -417,6 +423,10 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 	ctx.Data["PinnedIssues"] = pinned
 	ctx.Data["IsRepoAdmin"] = ctx.IsSigned && (ctx.Repo.IsAdmin() || ctx.Doer.IsAdmin)
 	ctx.Data["IssueStats"] = issueStats
+	ctx.Data["OpenCount"] = issueStats.OpenCount
+	ctx.Data["ClosedCount"] = issueStats.ClosedCount
+	ctx.Data["OpenLink"] = fmt.Sprintf("%s?q=%s&type=%s&sort=%s&state=open&labels=%s&milestone=%d&project=%d&assignee=%d&poster=%d&archived=%s", ctx.Link, keyword, viewType, sortType, selectLabels, mentionedID, projectID, assigneeID, posterID, showArchivedLabels)
```

Comment:
Not a blocker.
could you format this some how may be multiple line concate.  for better readability.

Reply:
> Not a blocker. could you format this some how may be multiple line concate or a function. for better readability.

Reply:
![image](https://github.com/go-gitea/gitea/assets/18380374/e39aa1a3-19dc-4f3a-9570-ef42ef5d9403)
Does this look good for you?

Reply:
Sorry for keeping you reiterating
https://github.com/go-gitea/gitea/pull/27302#discussion_r1346663806